integrations/Amplitude/browser.js

```diff
@@ -338,6 +338,10 @@ class Amplitude {
     this.setDeviceId(rudderElement);
 
     const { properties, name, category } = rudderElement.message;
+    const { integrations } = rudderElement.message;
+    let useNewPageEventNameFormat;
+    if (integrations && integrations.AM)
+      useNewPageEventNameFormat = integrations.AM.useNewPageEventNameFormat;
```

Comment:
You can re-factor to something simple like below,

```suggestion
    let useNewPageEventNameFormat = rudderElement.message.integrations?.AM?.useNewPageEventNameFormat || false;
```